src/s3contents.ts

```diff
@@ -1114,11 +1087,17 @@ export class Drive implements Contents.IDrive {
         );
         path = isDir ? path + '/' : path;
 
-        Contents.forEach(c => {
+        const promises = Contents.map(async c => {
```

Comment:
This function does not need to be `async` because `copy_file` already returns a promise.
```suggestion
        const promises = Contents.map(c => {
```